app/views/application/_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="col-lg-12">
+  <h2>New Movie</h2>
```

Comment:
Will this create/update form display with the header "New Movie" for each media type?